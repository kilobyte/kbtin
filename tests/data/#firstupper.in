#foreach {abc Abc ąbc Ąbc AOw Aow aOw aOw aów aÓw Ąów ĄÓw} {#firstu res {$0};#show $res}
