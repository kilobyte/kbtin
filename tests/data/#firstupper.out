Abc
Abc
Ąbc
Ąbc
AOw
Aow
AOw
AOw
Aów
AÓw
Ąów
ĄÓw
